### ISSUE TITLE
Can not use exec

### ISSUE THREAD

**Issue by kodypeterson** [jfromaniello/winser#35]
I am trying to use the following. It works when ran with "node app.js" but not from within the service:

var exec = require('child_process').execFile;
exec("C:\scripts\" + req.params.script + ".exe", function(error, stdout, stderr){
     console.log(arguments);
});

Any ideas?

**Comment by jmorrisIII** [comment]
Did you ever figure this out?

**Comment by kodypeterson** [comment]
Nope, still a problem

**Comment by jmorrisIII** [comment]
I think there is a problem with running an executable from a windows service after XP.

**Comment by kodypeterson** [comment]
Yeah, so req.params.script is the location to the executable. So lets just say this is the code failing:

var exec = require('child_process').execFile;
exec("C:\scripts\test.exe", function(error, stdout, stderr){
    console.log(arguments);
});

**Comment by deliverymanager** [comment]
Did you manage to figure this problem of the exec command not working to a .exe file?

**Comment by JBarna** [comment]
I found a reason for this problem at [this link](http://stackoverflow.com/questions/5307968/how-can-i-run-an-exe-program-from-a-windows-service-using-c). Turns our you can't create processes from windows services anymore.
